websites/A/AnimeDrive/metadata.json

```diff
@@ -6,20 +6,23 @@
 	},
 	"service": "AnimeDrive",
 	"description": {
-		"en": "AnimeDrive is an anime streaming website with Hungarian subs/dubs"
+		"en": "AnimeDrive is an anime streaming website with Hungarian subs/dubs",
+		"hu": "Az AnimeDrive egy anime streaming weboldal Magyar felirattal/szinkronnal"
 	},
 	"url": [
 		"animedrive.hu",
 		"www.animedrive.hu"
 	],
-	"version": "1.0.1",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/A/AnimeDrive/assets/logo.png",
-	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/A/AnimeDrive/assets/thumbnail.png",
+	"version": "1.0.2",
+	"logo": "https://cdn.discordapp.com/icons/702098578251841577/be50b5dbcb30bd99512b6df29284db0c.webp",
+	"thumbnail": "https://i.imgur.com/uP6s5yc.png",
 	"color": "#6f0094",
 	"category": "anime",
 	"tags": [
 		"anime",
 		"animes",
 		"animedrive"
-	]
-}
\ No newline at end of file
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*"
```

Comment:
If possible, make this regex more specific to the iframe hosts the site uses. If there are too many, or they change too often, you may disregard this.

Reply:
It usually changes and that's the only one on the site so I can't make it more specific..